app/components/__test__/Button.test.tsx

```diff
@@ -0,0 +1,61 @@
+import { fireEvent, render } from '@testing-library/react';
+
+import { Button, LinkButton } from '../Button';
+
+describe('Button', () => {
+  const btnText = 'Click me';
+  it('renders', () => {
+    const { getByText } = render(
+      <Button size="small" label="Click me" varient="primary" disabled={false} />
```

Comment:
you can replace label to use the variable `btnText`, i.e `label={btnText}`

Reply:
Addresssed